src/visualization/types/Gauge/options.tsx

```diff
@@ -1,141 +1,98 @@
-                    placeholder="Enter a number"
-                    onChange={evt => {
-                      setDigits(convertUserInputToNumOrNaN(evt))
-                    }}
-                    value={properties.decimalPlaces.digits}
-                    min={MIN_DECIMAL_PLACES}
-                    max={MAX_DECIMAL_PLACES}
-                    type={InputType.Number}
-                  />
-                }
+export const GaugeOptions: FC<Props> = ({properties, update}) => (
+  <Grid>
+    <Grid.Row>
+      <Grid.Column>
+        <Grid.Row>
+          <Grid.Column widthXS={Columns.Six}>
+            <Form.Element label="Value Prefix">
+              <Input
+                testID="prefix-input"
+                value={properties.prefix}
+                onChange={evt => {
+                  update({prefix: evt.target.value})
+                }}
+                placeholder="%, MPH, etc."
               />
             </Form.Element>
-          )}
-        </Grid.Column>
-        <Grid.Column widthXS={Columns.Twelve} widthMD={Columns.Six}>
-          <Form.Element label="Colorized Thresholds">
-            <ThresholdsSettings
-              thresholds={properties.colors}
-              onSetThresholds={colors => {
-                update({colors})
-              }}
-            />
-          </Form.Element>
-        </Grid.Column>
-      </Grid.Row>
-    </Grid>
-  )
-}
-
-export default GaugeOptions
+          </Grid.Column>
+          <Grid.Column widthXS={Columns.Six}>
+            <Form.Element label="Value Suffix">
+              <Input
+                testID="suffix-input"
+                value={properties.suffix}
+                onChange={evt => {
+                  update({suffix: evt.target.value})
+                }}
+                placeholder="%, MPH, etc."
+              />
+            </Form.Element>
+          </Grid.Column>
+        </Grid.Row>
+        <Grid.Row>
+          <Grid.Column widthXS={Columns.Six}>
+            <Form.Element label="Axis Prefix">
+              <Input
+                testID="tick-prefix-input"
+                value={properties.tickPrefix}
+                onChange={evt => {
+                  update({tickPrefix: evt.target.value})
+                }}
+                placeholder="%, MPH, etc."
+              />
+            </Form.Element>
+          </Grid.Column>
+          <Grid.Column widthXS={Columns.Six}>
+            <Form.Element label="Axis Suffix">
+              <Input
+                testID="tick-suffix-input"
+                value={properties.tickSuffix}
+                onChange={evt => {
+                  update({tickSuffix: evt.target.value})
+                }}
+                placeholder="%, MPH, etc."
+              />
+            </Form.Element>
+          </Grid.Column>
+        </Grid.Row>
+        {properties.decimalPlaces && (
+          <DecimalPlaces
+            isEnforced={properties?.decimalPlaces?.isEnforced ?? false}
```

Comment:
can this be simplified to

```ts
properties?.decimalPlaces?.isEnforced === true
```

so we don't need to worry about fallback behavior?

Reply:
Oh nice, yeah that is both cool & concise while also very easy to understand!